led_controll.py

```diff
@@ -0,0 +1,213 @@
+import RPi.GPIO as GPIO
+import time
+
+parking1_detected = True
+parking2_detected = True
+parking3_detected = True
+parking4_detected = True
+parking5_detected = True
+parking6_detected = True
+
+GPIO.setmode(GPIO.BOARD)
+GPIO.setwarnings(False)
+
+# define pins
+parking1green = 1
+parking1red = 2
+parking2green = 3
+parking2red = 4
+parking3green = 5
+parking3red = 6
+parking4green = 7
+parking4red = 8
+parking5green = 9
+parking5red = 10
+parking6green = 11
+parking6red = 12
+
+# setup
+GPIO.setup(parking1green, GPIO.OUT)
+GPIO.setup(parking1red, GPIO.OUT)
+GPIO.setup(parking2green, GPIO.OUT)
+GPIO.setup(parking2red, GPIO.OUT)
+GPIO.setup(parking3green, GPIO.OUT)
+GPIO.setup(parking3red, GPIO.OUT)
+GPIO.setup(parking4green, GPIO.OUT)
+GPIO.setup(parking4red, GPIO.OUT)
+GPIO.setup(parking5green, GPIO.OUT)
+GPIO.setup(parking5red, GPIO.OUT)
+GPIO.setup(parking6green, GPIO.OUT)
+GPIO.setup(parking6red, GPIO.OUT)
+
+
+# functions turn on green leds
+def turn_on_parking1green():
+    GPIO.output(parking1green, GPIO.HIGH)

```

Comment:
This sort of functions can be rewritten to only define one function, where the  parking lot number and pins are given, like so:
```python
def turn_on_green(pin_no: int) -> None:
    GPIO.output(pin_no, GPIO.HIGH) 
```